pkg/util/util.go

```diff
@@ -430,13 +431,15 @@ func ResolvePVCFromPod(pod *corev1.Pod, pvcLister corelisterv1.PersistentVolumeC
 			pvc, err := pvcLister.PersistentVolumeClaims(pod.Namespace).Get(pvcName)
 			if err != nil {
 				klog.Errorf("Get PVC %s/%s error: %v", pod.Namespace, pvcName, err)
-				continue
+				return nil, err
 			}
 			pvcs = append(pvcs, pvc)
 		}
 	}
 	if len(pvcs) == 0 {
-		return nil, errors.NewNotFound(corev1.Resource("pvc"), pod.Name)
+		err := errors.NewNotFound(corev1.Resource("pvc"), "")
```

Comment:
Refer to the comment here https://github.com/pingcap/tidb-operator/pull/3816/files#r587978537.

Reply:
details are fixed in the next line